public/auth.html

```diff
@@ -0,0 +1,32 @@
+<html>
+<head>
+<title>Login to AVF Engagement & Media Dashboard</title>
+    <link rel="shortcut icon" type="image/png" href="favicon.png"/>
+    <link rel="stylesheet" type="text/css" href="styles.css">
+    <script src="https://www.gstatic.com/firebasejs/5.5.9/firebase-app.js"></script>
+    <script src="https://www.gstatic.com/firebasejs/5.6.0/firebase-auth.js"></script>
+    <script src='firebase.js'></script>
+    <script src="https://cdn.firebase.com/libs/firebaseui/3.4.1/firebaseui.js"></script>
+    <link type="text/css" rel="stylesheet" href="https://cdn.firebase.com/libs/firebaseui/3.4.1/firebaseui.css" />
+    <script>
+        // FirebaseUI config.
+        var uiConfig = {
+            signInSuccessUrl: "index.html",
```

Comment:
I have changed this to point to index page instead of having two auth pages one pointing to coding progress page and the other to one of the media engagement page.

Reply:
thanks